### PR TITLE
profile form right hand side

### PR DESCRIPTION
**Pull request by tundeloper**
# *Pull Request Template*

## Working on profile section right-hand-side

this changes add ```profile``` route to the route file

## Related Issue(s)

- [ ] Issue 1
- [ ] Issue 2

## Checklist

- [x] I have tested the changes locally and they work as expected.
- [x] The code follows the project's coding standards and conventions.
- [x] I have added appropriate comments and documentation where necessary.
- [x] All tests passed successfully.
- [x] I have updated the necessary dependencies.
- [x] I ran ``` npm run build ``` and it built without errors

## Changes Made

Please provide a list of the main changes made in this pull request.
added profile routes,
added css file to the style directory
added settings svg to the public directory
added profile background image to the public directory 

## Screenshots

If applicable, add screenshots or GIFs to showcase the changes visually.

## Dependencies or File Changes
src/route
src/pages/profile.jsx
src/styles/profile.css
public directory

# Please list any new dependencies or files added or updated in this pull request.

src/profile
public/bgimage.png
public/settingSvg.jsx

## Testing Instructions

Please provide clear instructions on how to test the changes made in this pull request.
go to /profile to see the changes made

### CODE REVIEW

File: src/components/header.jsx
Comment (Christiano112) [inline src/components/header.jsx:1]: Is this the updated content of the header file?

File: src/components/loading.jsx
Comment (Christiano112) [inline src/components/loading.jsx:1]: Delete the comments

File: src/pages/home.jsx
Comment (Christiano112) [inline src/pages/home.jsx:1]: I don't think your branch is updated.... kindly review your work

**Review by Christiano112 — CHANGES_REQUESTED** [review]

File: src/pages/home.jsx
Comment (tundeloper) [reply src/pages/home.jsx:1]: okay

**Comment by Christiano112** [comment]
Your codebase isn't updated with the current code on develop

**Pull request closed without merging**